CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -606,9 +627,6 @@ private string ReadUrlFromDragDropData(IDataObject data, string urlDataFormatNam
             return url.TrimEnd('\0');
         }
 
-
```

Comment:
Are you suggesting I put the white space back? (I did remove that extraneous white space, yes.)